frontend/shared-components/logo.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react"
+import LogoSVG from '../public/NPDCLogo.svg'
+
+export default function Logo() {
```

Comment:
should likely have only one place with all logos.

Reply:
I believe this is the only one so far - are you seeing somewhere else with logos?